services/src/main/scala/cromwell/pipeline/service/ProjectConfigurationService.scala

```diff
@@ -0,0 +1,24 @@
+package cromwell.pipeline.service
+
+import cromwell.pipeline.datastorage.dao.repository.DocumentRepository
+import cromwell.pipeline.datastorage.dto.{ ProjectConfiguration, ProjectId }
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class ProjectConfigurationService(repository: DocumentRepository)(implicit ec: ExecutionContext) {
+  import ProjectConfiguration._
+
+  def addConfiguration(projectConfiguration: ProjectConfiguration): Future[String] =
+    repository
+      .updateOne(toDocument(projectConfiguration), "projectId", projectConfiguration.projectId.value)
+      .map(_.toString)
+
+  def getById(projectId: ProjectId): Future[Option[ProjectConfiguration]] =
+    repository.getByParam("projectId", projectId.value).map {
```

Comment:
```suggestion
    repository.getByParam("projectId", projectId.value).headOption.map(fromDocument)
```

Reply:
`Seq` doesn't have `headOption` :(

Reply:
What? :)
`headOption` is a method from `GenTraversableLike` trait, which is almost the highest trait in the Scala collections hierarchy. Almost every scala collection has this method.
In particular, `Seq` does have this method.